flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/dataset/BatchScan.scala

```diff
@@ -38,7 +39,9 @@ trait BatchScan extends CommonScan with DataSetRel {
 
     val inputType = input.getType
 
-    val internalType = FlinkTypeFactory.toInternalRowTypeInfo(getRowType)
```

Comment:
These changes can be reverted if we leave the `FlinkTypeFactory.toInternalRowTypeInfo()` method as it is.